community/sigs.md

```diff
@@ -0,0 +1,15 @@
+# SIGs - Special Interest Groups
+
+As described in the ONNX [governance](https://github.com/onnx/onnx/tree/master/community#sig---special-interest-groups), Special Interest Groups (SIGs) are persistent groups responsible for specific parts of the project. SIGs have open and transparent proceedings to develop goals and implement code contributions. SIGs are also responsible for ongoing maintenance of the code in their areas.
+
+## Joining a SIG
+If you are interested in participating, please join the discussion in the respective Gitter room. Details about any upcoming meetings will also be shared in the Gitter rooms. SIG artifacts can be found in the [sigs repository](https://github.com/onnx/sigs).
```

Comment:
Could be good to add the gitter link so users can click on the hyperlink in the "joining a SIC" section ( https://gitter.im/onnx)